lgc/include/lgc/patch/Patch.h

```diff
@@ -73,7 +73,7 @@ class LegacyPatchCheckShaderCache;
 // Initialize passes for patching
 //
 // @param passRegistry : Pass registry
-inline static void initializePatchPasses(llvm::PassRegistry &passRegistry) {
+inline static void initializePatchPasses(llvm::PassRegistry &passRegistry) { // NOLINT
```

Comment:
The lint here is weird; the comments don't seem to silence it for whatever reason. I think it may need something like [`[[gnu::used]]`](https://clang.llvm.org/docs/AttributeReference.html#used), but I'd just ignore it instead

Reply:
Ah, I know why. This seems to be because this is a free function declared as `static`. I think removing `static` will resolve this.